src/sentry/assistant/guides.py

```diff
@@ -20,4 +20,6 @@
     "issue_stream": {"id": 3, "required_targets": ["issue_stream"]},
     "inbox_guide": {"id": 8, "required_targets": ["inbox_guide"]},
     "for_review_guide": {"id": 9, "required_targets": ["for_review_guide_tab"]},
+    "alerts_write_member": {"id": 10, "required_targets": ["alerts_write_member"]},
+    "alerts_write_owner": {"id": 11, "required_targets": ["alerts_write_owner"]},
```

Comment:
We should maintain a list of already used IDs above this

Reply:
makes sense, were you thinking a commented list for record keeping purposes or some kind of list and business logic to ensure no ids were reused?